pages.it/common/rm.md

```diff
@@ -17,7 +17,7 @@
 
 - Rimuovi file interattivamente, chiedendo conferma prima di rimuovere ogni file:
 
-`rm -i {{file(s)}}`
+`rm -i {{file1 file2 ...}}`
```

Comment:
```suggestion
`rm -i {{path/to/file1 path/to/file2 ...}}`
```

Mandatory change: [path format is wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): `path/to/` is expected but `file` is met.

Reply:
As far as I know, `path/to/` is not correct Italian.

Reply:
> As far as I know, `path/to/` is not correct Italian.

Yeah, I think we can use the one from common arguments.

Reply:
Suggestion updated according to google translate.